AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/actions.js

```diff
@@ -67,7 +64,7 @@ define([
                 return;
             }
 
-            this.rows().forEach(function (record) {
+            this.source().data.items.forEach(function (record) {
                 if (record.id === displayedRecord.id) {
                     updatedDisplayedRecord = record;
```

Comment:
Can add the `break;` here? As we got the displayedRecord then I don't think we need to run the whole loop? Just suggestion.